app/repositories/user_repository.py

```diff
@@ -1,31 +1,48 @@
 import app.mappers.user_mapper as mapper
 from app import db
 from app.dtos.user_dto import UserDTO
+from app.exceptions import UserNotExistError
 from app.models.user import User
 
 
-def create_new_user(user_dto: UserDTO):
-    new_user = mapper.of_dto(user_dto)
-    db.session.add(new_user)
-    db.session.commit()
-
-
-def get_user_by_id(user_id: int) -> UserDTO:
-    user = User.query.filter_by(id=user_id).first()
-    return mapper.to_dto(user)
-
-
-def get_user_by_email(email: str) -> UserDTO:
-    user = User.query.filter_by(email=email).first()
-    return mapper.to_dto(user)
-
-
-def delete_user_by_id(user_id: int) -> None:
-    User.query.filter_by(id=user_id).delete()
-    db.session.commit()
-
-
-def update_user_password(user_id: int, new_password: str) -> None:
-    user = User.query.filter_by(id=user_id).first()
-    user.password = new_password
-    db.session.commit()
+class UserRepository:
```

Comment:
Czemu te metody sa staryczne?

Reply:
bo w pythonie klasy słuzą do dekompozycji kodu, a nie programowania obiektowego w 90% przypadków i tak właśnie tutaj jest to zrobione, tak naprawdę @staticmethod nic nie zmienia w działaniu metody, a jest to nic innego jak konwencja do oznaczania